docs/output.md

```diff
@@ -6,554 +6,671 @@ This document describes the output produced by the pipeline. Most of the plots a
     - [Manta](#manta)
     - [TIDDIT](#tiddit)
-    - [Sentieon DNAscope SV](#sentieon-dnascope-sv)
   - [Sample heterogeneity, ploidy and CNVs](#sample-heterogeneity-ploidy-and-cnvs)
     - [ASCAT](#ascat)
+    - [CNVKit](#cnvkit)
     - [Control-FREEC](#control-freec)
   - [MSI status](#msi-status)
-    - [MSIsensor](#msisensor)
+    - [MSIsensorPro](#msisensorpro)
 - [Variant annotation](#variant-annotation)
   - [snpEff](#snpeff)
   - [VEP](#vep)
-- [QC and reporting](#qc-and-reporting)
-  - [QC](#qc)
+- [Quality control and reporting](#qc-and-reporting)
+  - [Quality control](#quality-control)
     - [FastQC](#fastqc)
-    - [bamQC](#bamqc)
+    - [FastP](#fastp)
     - [GATK MarkDuplicates reports](#gatk-markduplicates-reports)
+    - [mosdepth](#mosdepth)
     - [samtools stats](#samtools-stats)
     - [bcftools stats](#bcftools-stats)
-    - [VCFtools](#vcftools)
+    - [vcftools](#vcftools)
     - [snpEff reports](#snpeff-reports)
     - [VEP reports](#vep-reports)
   - [Reporting](#reporting)
     - [MultiQC](#multiqc)
   - [Pipeline information](#pipeline-information)
+- [Reference files](#reference-files)
+
+## Directory Structure
+
+The default directory structure is as follows
+
+```
+{outdir}
+├── csv
+├── multiqc
+├── pipeline_info
+├── preprocessing
+│   └── <sample>
+│       ├── markduplicates
+│       ├── recal_table
+│       └── recalibrated
+├── reference
+└── reports
+    ├── <tool1>
+    └── <tool2>
+work/
+.nextflow.log
+```
 
 ## Preprocessing
 
-`Sarek` pre-processes raw `FASTQ` files or `unmapped BAM` files, based on [GATK best practices](https://gatk.broadinstitute.org/hc/en-us/sections/360007226651-Best-Practices-Workflows).
+Sarek pre-processes raw FastQ files or unmapped BAM files, based on [GATK best practices](https://gatk.broadinstitute.org/hc/en-us/sections/360007226651-Best-Practices-Workflows).
 
-### Map to Reference
+### Preparation of input files (FastQ or (u)BAM)
 
-#### BWA
+[FastP](https://github.com/OpenGene/fastp) is a tool designed to provide all-in-one preprocessing for FastQ files and as such is used for trimming and splitting. By default, these files are not published. However, if publishing is enabled, please be aware that these files are only published once, meaning if trimming and splitting is enabled, then the resulting files will be sharded FastQ files with trimmed reads. If only one of them is enabled then the files contain either trimmed or split reads, respectively.
 
-[BWA](https://github.com/lh3/bwa) is a software package for mapping low-divergent sequences against a large reference genome.
+#### Trim adapters
 
-Such files are intermediate and not kept in the final files delivered to users.
+[FastP](https://github.com/OpenGene/fastp) supports global trimming, which means it trims all reads in the front or the tail. This function is useful since sometimes you want to drop some cycles of a sequencing run. In the current implementation in sarek
```

Comment:
```suggestion
[FastP](https://github.com/OpenGene/fastp) supports global trimming, which means it trims all reads in the front or the tail. This function is useful since sometimes you want to drop some cycles of a sequencing run. In the current implementation in Sarek
```

At least everywhere else it's capitalized.